submissions/BlueLamantine/memory-pair-game/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hairy Pawtter</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/map.css" />
+    <link rel="stylesheet" href="css/paws.css" />
+    <link rel="stylesheet" href="css/cards.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" href="img/cursor_pointer.png" />
+  </head>
+  <body>
+    <header class="main-header">
+      <h1>Hairy Pawtter</h1>
+      <h5>Memory-pair game</h5>
+    </header>
+    <nav class="panel">
+      <div id="moves-counter" class="moves"><span id="counter">0</span> Moves</div>
+      <div id="timer">00 : 00 : 00</div>
+      <div class="music">
```

Comment:
I'd recommend using the `button` tag here. It will be more semantically correct

Reply:
ok